postman/patient-demographics-sandbox.postman_collection.json

```diff
@@ -1000,57 +1498,39 @@
 					},
 					"response": []
 				},
-				{
-					"name": "Search with limited results",
-					"request": {
-						"method": "GET",
-						"header": [
-							{
-								"key": "X-Request-ID",
-								"value": "{{$guid}}",
-								"type": "text"
-							}
-						],
-						"url": {
-							"raw": "{{base_url}}/Patient?family=Sm*&gender=female&birthdate=eq2010-10-22&_max-results=1",
-							"host": [
-								"{{base_url}}"
-							],
-							"path": [
-								"Patient"
-							],
-							"query": [
-								{
-									"key": "family",
-									"value": "Sm*"
-								},
-								{
-									"key": "gender",
-									"value": "female"
-								},
-								{
-									"key": "birthdate",
-									"value": "eq2010-10-22"
-								},
-								{
-									"key": "_max-results",
-									"value": "1"
-								}
-							]
-						},
-						"description": "Example of a wildcarded search with limited results, where the \"family\" name is wildcarded and \"_max-results\" is added, which limits the number of results to 2 patients - will return a result of 2 patients - Jane Smith patient (nhsId - 9000000009) & Jayne Smith (nhsId - 9000000017)\r\n\r\nNote: if \"_max-results\" is reduced to 1 then a \"TOO_MANY_RESULTS\" error response will be returned, as the number of matched patients found is greater than the \"_max-results\" parameter."
-					},
-					"response": []
-				},
```

Comment:
Why was this scenario removed?

Reply:
This is now "Wildcard search with results limit" - since it's a wildcard search, I gave it a more accurate name